test/iron-overlay-behavior.html

```diff
@@ -421,6 +434,45 @@
 
       });
 
+      suite('multiple overlays with backdrop implemented in different elements', function () {
+        var overlays;
+
+        setup(function() {
+          overlays = fixture('backdrop-different-elements');
+        });
+
+        test('multiple overlays share the same backdrop', function() {
+          assert.equal(overlays[0].backdropElement, overlays[1].backdropElement);
+        });
+
+        test('multiple overlays close the corrent backdrop', function(done) {
```

Comment:
I actually meant "correct". But since we are only talking about one single backdrop now maybe it should just be "the backdrop"